aboutMe.markdown

```diff
@@ -66,7 +66,8 @@ My most popular open-source projects, sorted by stars:
  * PHP language trivia [PHPKonf'17] <br>
    ([slides](https://www.slideshare.net/nikita_ppv/php-language-trivia))
  * Static Optimization of PHP bytecode [PHPSC'17] <br>
-   ([slides](https://www.slideshare.net/nikita_ppv/static-optimization-of-php-bytecode-phpsc-2017))
+   ([slides](https://www.slideshare.net/nikita_ppv/static-optimization-of-php-bytecode-phpsc-2017),
+   [video](https://vimeo.com/237704382))
```

Comment:
This is a different conference (phpday'17), so should be a separate entry :)

Reply:
Same slides or no?